### PR TITLE
Update permissions.yml star sapphire CL

### PR DESCRIPTION
**Pull request by ViceTOP**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -5396,6 +5396,33 @@ groups:
         - ch.alias.haste0
         - ch.alias.haste1
         - ch.alias.haste2
+  starsapphirecarolferris:
+    options:
+      prefix: '&dSta&5r&7_&dS&5a&dpp&5hir&de&7_&dC&5F'
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: should be no underscores in the prefix

**Comment by malonnnn** [comment]
if this is in relation to green lantern, then it should be

greenlanternstarsapphirecarolferris
Green Lantern-SS
Green Lantern-CF
Green Lantern-SSCF

**Comment by ViceTOP** [comment]
It isnt a green lantern, star sapphires are their own corps

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -5396,6 +5396,33 @@ groups:
         - ch.alias.haste0
         - ch.alias.haste1
         - ch.alias.haste2
+  starsapphirecarolferris:
+    options:
+      prefix: '&dSta&5r&7 &dS&5a&dpp&5hir&de&7 &dC&5F'
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: need a dash (-) between Sapphire and CD

**Pull request merged**